src/Layout.scss

```diff
@@ -0,0 +1,3 @@
+.layout {
```

Comment:
Better not add this, we prefer not adding style unless necessary for the functionality, so developers can easily customize as they wish while keeping our code simple.